src/styles/youtubeMockup.scss

```diff
@@ -0,0 +1,295 @@
+//YouTube Mockup Styles
+.youtube-mockup {
+  background-color: $grayOne;
+}
+
+.main-content {
+  background-color: $grayTwo;
+}
+
+.header-container {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+//Browser Toolbar Styles
+.browser-toolbar {
+  display: flex;
```

Comment:
Alphabetize css rules:

This is a standard we follow at BizStream, and you don't have to change this everywhere now, just letting you know for the future. You might like the idea and add it to personal projects, or just use it moving forward cause it makes styles easier to read and find the style rule you are looking for. 

There are also IDE/linter settings you can set so that on save, it will automatically reformat it to be alphabetical for you.

Reply:
Again, don't worry about changing all of this now 😃